src/friendly-iframe-embed.js

```diff
@@ -368,10 +369,17 @@ export class FriendlyIframeEmbed {
       ? this.host.signals()
       : new Signals();
 
+    /** @private @const {!Deferred} */
+    this.renderComplete_ = new Deferred();
+
     /** @private @const {!Promise} */
     this.winLoadedPromise_ = Promise.all([loadedPromise, this.whenReady()]);
     if (this.ampdoc) {
-      this.whenReady().then(() => this.ampdoc.setReady());
+      // TODO(ccordry): wait for renderComplete after no signing launch.
+      const readyPromise = this.spec.skipHtmlMerge
+        ? this.whenRenderComplete()
+        : this.whenReady();
```

Comment:
Let's rename this function to avoid confusion. This should be `whenRenderStarted()`.

Reply:
Done.